src/prelude.rs

```diff
@@ -1,27 +1,17 @@
 //! Commonly used items.
 
-#![deprecated(note = "Use dispatching2 instead")]
-#![allow(deprecated)]
-
 pub use crate::{
     error_handlers::{LoggingErrorHandler, OnError},
     respond,
 };
 
 pub use crate::dispatching::{
-    dialogue::{
-        exit, next, DialogueDispatcher, DialogueStage, DialogueWithCx, GetChatId, Transition,
-        TransitionIn, TransitionOut,
-    },
-    Dispatcher, DispatcherHandlerRx, DispatcherHandlerRxExt, UpdateWithCx,
+    dialogue::Dialogue, Dispatcher, HandlerExt as _, MessageFilterExt as _, UpdateFilterExt as _,
 };
 
-#[cfg(feature = "macros")]
-pub use crate::teloxide;
```

Comment:
is this macro not used anymore?

Reply:
Yes, it will be deprecated in teloxide-macros.

Reply:
Shouldn't it just be deleted, do we need to keep it in deprecated state?